src/Microsoft.DotNet.Cli.Utils/Interop.cs

```diff
@@ -0,0 +1,85 @@
+﻿// This file is decompiled C# from an assembly generated by tlbimp against a copy of urlmon's idl file
+
+// UrlMonTypeLib.IInternetSecurityManager
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.DotNet.Cli.Utils
+{
+    [ComImport]
+    [Guid("79EAC9EE-BAF9-11CE-8C82-00AA004BA90B")]
+    [InterfaceType(ComInterfaceType.InterfaceIsIUnknown)]
+    [ComConversionLoss]
+    internal interface IInternetSecurityManager
+    {
+        [MethodImpl(MethodImplOptions.InternalCall)]
```

Comment:
I know this is copied, but I think the InternalCall is tlbimp bug and unnecessary.

Reply:
[MethodImpl(MethodImplOptions.InternalCall)]

Reply:
So just remove all [MethodImpl(MethodImplOptions.InternalCall)]?

Reply:
I removed all  [MethodImpl(MethodImplOptions.InternalCall)] still works